src/hooks/useCustomQuery.ts

```diff
@@ -43,7 +44,7 @@ export default function useCustomQuery<
 
   return useQuery<TQueryFnData, TError, TData, TQueryKey>({
     // TODO: Fix this type assertion
```

Comment:
Czy to nie zostało właśnie naprawione?

Reply:
W sumie to było wcześniej naprawione. Usunąłem teraz komentarz.